src/common/CurrencyIcon/index.tsx

```diff
@@ -17,6 +24,7 @@ interface CurrencyIconProps {
 export const CurrencyIcon = ({ currency, size = 24, ...props }: CurrencyIconProps): JSX.Element => {
   switch (currency) {
     case LBTC_TICKER:
+    case LBTC_TESTNET_TICKER:
```

Comment:
we have USDT and LCAD ticker for testnet :) 

https://blockstream.info/liquidtestnet/asset/ac3e0ff248c5051ffd61e00155b7122e5ebc04fd397a0ecbdd4f4e4a56232926

https://blockstream.info/liquidtestnet/asset/f3d1ec678811398cd2ae277cbe3849c6f6dbd72c74bc542f7c4b11ff0e820958